### PR TITLE
Dev/fix 3.1

### PR DESCRIPTION
**Pull request by pierrz**
Data input fix, review fixes, locks

### CODE REVIEW

File: harvester/__init__.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 import requests
 
-from .config import harvester_config
```
Comment (pierrz) [inline harvester/__init__.py:11]: @geeky-do those (not relative) imports, I need to understand why they don't work for the tests ... that's the only issue remaining I think (tests still pass in github-actions despite this)

File: api/test/conftest.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 import pytest
 from fastapi.testclient import TestClient
-from main import app
+from api.main import app
```
Comment (pierrz) [inline api/test/conftest.py]: @geeky-do same import issue mystery here and other files of the API

File: harvester/__init__.py (hunk 1 above)
Comment (geeky-do) [reply harvester/__init__.py:11]: you need to set something known as PYTHONPATH that tells python where is the root folder for your code so let's say you are using pytest you can add this to your project.toml:

`[tool.pytest.ini_options]
pythonpath = [
   "path/to/main",
]`

In your code base, I would suggest you always use full imports and not relative ones see [this](https://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module)

It's just a better practice in general I would say.

Comment (pierrz) [reply harvester/__init__.py:11]: Much better thank you !

**Pull request merged**